roles/download/defaults/main.yml

```diff
@@ -263,6 +277,20 @@ dashboard_image_repo: "gcr.io/google_containers/kubernetes-dashboard-{{ image_ar
 dashboard_image_tag: "v1.10.1"
 
 downloads:
+  crictl:
+    container: false
+    file: true
+    enabled: true
```

Comment:
crictl should not be downloaded always.
Some of the download files has condition.
Please check it.